src/testapp/models.py

```diff
@@ -82,8 +82,33 @@ class SimpleExcludeModel(models.Model):
     history = AuditlogHistoryField()
 
 
+class AdditionDataIncludedModel(models.Model):
```

Comment:
I guess this is a typo. It does not break anything, but I think this class should be named ```AdditionalDataIncludedModel```.